Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/HOKSAMLAssertionBuilder.java

```diff
@@ -690,4 +707,17 @@ public static Evidence buildEvidence(String evAssertionID, DateTime issueInstant
     private static String createAssertionId() {
         return ID_PREFIX.concat(String.valueOf(UUID.randomUUID())).replaceAll("-", "");
     }
+
+    protected boolean isAuthDEvidenceConditionsDefaultValueEnabled() {
+        //if not provided or invalid return true else false
+        try {
+            String authDEvidenceConditionsDefaultValueEnabled = PropertyAccessor.getInstance().getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.ENABLE_AUTH_DEC_EVIDENCE_CONDITIONS_DEFAULT_VALUE);
+            if (authDEvidenceConditionsDefaultValueEnabled != null) {
+                return !authDEvidenceConditionsDefaultValueEnabled.equals(Boolean.FALSE.toString());
```

Comment:
I think the value in the property file should be true so noone has to do any updates when they upgrade but if someone accidentally changed it to property=crapValue, I think it's right to default to not use the default values.  We agreed that it's better to not use the values is best practice.  By setting it to true in the prop file it allows that no real upgrades are needed for the adopter that is upgrading.  Hopefully that makes sense.